app/src/main/java/fr/gaulupeau/apps/Poche/ui/Themes.java

```diff
@@ -44,7 +45,7 @@ public static void applyDialogTheme(Activity activity) {
 
     public static void checkTheme(Activity activity) {
         Theme appliedTheme = appliedThemes.get(activity);
-        if(appliedTheme != theme) activity.recreate();
+        if(appliedTheme != theme) ActivityCompat.recreate(activity);
```

Comment:
Probably needs to be tested on <28 API.

Reply:
Hmm... I don't have any such device, and I cannot test with an emulator...

Reply:
I'll try to test it myself later.